SU-SWS/stanford_paragraph_types
# HSDO-952 reduced margin bottom for callout text with colored background

# READY FOR REVIEW

# Summary
- Remove extra padding from callout text when background color is applied

# Needed By (Date)
- August 2, 2017

# Urgency
- low

# Steps to Test

1. checkout branch HSDO-952
2. Add Callout Text paragraph type to Story Page content type. Ensure that there is a reduced bottom margin.

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
